### PR TITLE
Ensure Geth is configured with sufficient block gas limit

### PR DESCRIPTION
**Pull request by cfelde**

### CODE REVIEW

**Comment by cfelde** [comment]
Upped the gas limit because we changed the default limit in web3j, Geth in line with Besu

File: src/main/resources/geth/dev.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "nonce": "0x0",
   "timestamp": "0x5db7096a",
   "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000fe3b557e8fb62b89f4916b721be55ceb828dbd730000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
-  "gasLimit": "0x47b760",
+  "gasLimit": "0x1fffffffffffff",
```
Comment (iikirilov) [inline src/main/resources/geth/dev.json]: can we update this for besu too please

Comment (cfelde) [reply src/main/resources/geth/dev.json]: Besu already on this value, so all good

**Comment by cfelde** [comment]
who can merge?

**Comment by iikirilov** [comment]
> who can merge?

I can

**Pull request merged**